### PR TITLE
Adding jwt-cpp module.

### PR DESCRIPTION
**Pull request by olipauly**

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (jwt-cpp) have been updated in this PR. Please review the changes.

File: modules/jwt-cpp/0.7.0/patches/add-workspace.patch
Comment (udaya2899) [inline modules/jwt-cpp/0.7.0/patches/add-workspace.patch:1]: You don't need a WORKSPACE file anymore

File: modules/jwt-cpp/0.7.0/source.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "url": "https://github.com/Thalhammer/jwt-cpp/archive/refs/tags/v0.7.0.tar.gz",
+    "integrity": "sha256-uesnDjuoIh5LK8OHI8mhy0+mwkGkKQi5ozTa/zETdAY=",
+    "strip_prefix": "jwt-cpp-0.7.0",
+    "patches": {
+        "add-build.patch": "sha256-J25yu7qPy4L8KkBnhzb3OTmvga0Gg8XIGD8Zh2L1D1U=",
+        "add-example-build.patch": "sha256-Y3pX2jEysCW+oLC7DXUOOM8qS3079y/pH5XtKfvD4I8=",
+        "add-include-jwt_cpp-build.patch": "sha256-S+qybqIQBE2KupJzvOH+j7HyBktJvRbJrj4MOFZ1sHg=",
+        "add-include-jwt_cpp-traits-nlohmann_json-build.patch": "sha256-mUQo0+2UAs3Ccr1+BBNOvH8UvSDhO0bNYED56mpuoks=",
+        "add-module_dot_bazel.patch": "sha256-asaiWINrAsuLY3QyXDRMGGaawxqGvRagKRElnCi08IU=",
+        "add-workspace.patch": "sha256-6DzTOqAlMMs4kdJWLQiVzmElb+woXNfMLs1pHgtb2Ig=",
```
Comment (udaya2899) [inline modules/jwt-cpp/0.7.0/source.json:11]: also remove

File: modules/jwt-cpp/0.7.0/patches/add-include-jwt_cpp-build.patch [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
++    features = ["-use_header_modules"],
++    include_prefix = "jwt-cpp",
++    deps = [
++        ":base64",
++        "@boringssl//:crypto",
++    ],
++    visibility = ["//visibility:public"],
++)
++
++build_test(
```
Comment (udaya2899) [inline modules/jwt-cpp/0.7.0/patches/add-include-jwt_cpp-build.patch:37]: We anyways add jwt to the presubmit.yml so the build_test here is redundant

**Review by udaya2899 — APPROVED** [review]

**Review by udaya2899 — APPROVED** [review]

**Review by meteorcloudy — APPROVED** [review]

**Pull request merged**